tools/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ install(FILES pibridge-shutdown.8
   DESTINATION ${CMAKE_INSTALL_MANDIR}/man8/
 )
 
-install(PROGRAMS ks8851-set-mac lan95xx-set-mac revpi-set-mac
+install(PROGRAMS ks8851-set-mac lan95xx-set-mac lan78xx-set-mac revpi-set-mac
```

Comment:
Another inconsistent mix of 7800 and 78xx. Can we have the same number everywhere instead please?

Reply:
Likewise, lan7800 comes before lan9514 in the "Supported devices types" message below, but here the order is reversed. Using the same order everywhere would be neater.

Reply:
> Another inconsistent mix of 7800 and 78xx. Can we have the same number everywhere instead please?

Same problem as above. The script is meant to be used with lan78xx devices, but the eeprom default image is specific to lan7800. So the naming is immo right